### PR TITLE
Separate Configs

### PR DESCRIPTION
**Pull request by manosim**

### CODE REVIEW

**Comment by JakeSidSmith** [comment]
o___O

File: packages/eslint-config-dabapps-base-react-native/package.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/dabapps/eslint-config"
+  },
+  "keywords": [
+    "eslint",
+    "eslintconfig",
+    "config",
+    "airbnb",
```
Comment (RealOrangeOne) [inline packages/eslint-config-dabapps-base-react-native/package.json]: um no

File: package.json [hunk 2]
```diff
@@ -14,6 +14,7 @@
     "eslint",
     "linting",
     "react",
+    "react-native",
```
Comment (RealOrangeOne) [inline package.json]: we really don't need keywords on this, it's private

File: packages/eslint-config-dabapps-react/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  },
+  "devDependencies": {
+    "eslint": "^3.9.1",
+    "eslint-find-rules": "^1.14.3",
+    "eslint-plugin-import": "^2.1.0",
+    "eslint-plugin-jsx-a11y": "^2.2.3",
+    "eslint-plugin-react": "^6.6.0",
+  },
+  "peerDependencies": {
+    "eslint": "^3.9.1",
```
Comment (RealOrangeOne) [inline packages/eslint-config-dabapps-react/package.json]: pin plz

File: package.json (hunk 2 above)
Comment (manosim) [inline package.json]: It is a public repository not prive.

Comment (RealOrangeOne) [inline package.json]: Yes, but it's not a public npm module. We should add "private": true so one of us doesnt accidently publish it to npm

**Comment by RealOrangeOne** [comment]
Ah ok. Makes sense. Can the directory names not be a lot be a lot shorter. Just 'react-native', 'react' etc?

File: eslint-config-dabapps-react-native/package.json [hunk 4]
```diff
@@ -0,0 +1,36 @@
+{
```
Comment (RealOrangeOne) [inline eslint-config-dabapps-react-native/package.json]: if we remove all these package.jsons, we can make it much cleaner. So in the project we just do 
`
"extends": {
    "eslint-config-dabapps/react-native"
}
`which is so much nicer!

Comment (JakeSidSmith) [inline eslint-config-dabapps-react-native/package.json]: Great minds think like Jake. 😉

**Comment by JakeSidSmith** [comment]
What if they decide to change their defaults?

**Comment by RealOrangeOne** [comment]
Any kind of review process go into this?...

**Pull request merged**